contributing-docs/08_static_code_checks.rst

```diff
@@ -579,118 +302,48 @@ command (example for ``airflow`` files):
--------------------------------------
-
-The static code checks can be launched using the Breeze environment.
-
-You run the static code checks via ``breeze static-check`` or commands.
-
-You can see the list of available static checks either via ``--help`` flag or by using the autocomplete
-option.
-
-Run the ``mypy`` check for the currently staged changes (in ``airflow/`` excluding providers):
-
-.. code-block:: bash
-
-     breeze static-checks --type mypy-airflow
-
-Run the ``mypy`` check for all files:
-
-.. code-block:: bash
-
-     breeze static-checks --type mypy-airflow --all-files
-
-Run the ``ruff`` check for the ``tests/core.py`` file with verbose output:
-
-.. code-block:: bash
-
-     breeze static-checks --type ruff --file tests/core.py --verbose
-
-Run the ``ruff`` check for the ``tests.core`` package with verbose output:
-
-.. code-block:: bash
-
-     breeze static-checks --type ruff --file tests/core/* --verbose
-
-Run the ``ruff-format`` check for the files ``airflow/example_dags/example_bash_operator.py`` and
-``airflow/example_dags/example_python_operator.py``:
-
-.. code-block:: bash
-
-     breeze static-checks --type ruff-format --file airflow/example_dags/example_bash_operator.py \
-         airflow/example_dags/example_python_operator.py
-
-Run all checks for the currently staged files:
-
-.. code-block:: bash
-
-     breeze static-checks
-
-Run all checks for all files:
-
-.. code-block:: bash
-
-    breeze static-checks --all-files
-
-Run all checks for last commit:
-
-.. code-block:: bash
-
-     breeze static-checks --last-commit
-
-Run all checks for all changes in my branch since branched from main:
-
-.. code-block:: bash
-
-     breeze static-checks --type mypy-airflow --only-my-changes
-
-More examples can be found in
-`Breeze documentation <../dev/breeze/doc/03_developer_tasks.rst#running-static-checks>`_
-
-
-Debugging pre-commit check scripts requiring image
+Debugging prek check scripts requiring image
 --------------------------------------------------
 
 Those commits that use Breeze docker image might sometimes fail, depending on your operating system and
 docker setup, so sometimes it might be required to run debugging with the commands. This is done via
 two environment variables ``VERBOSE`` and ``DRY_RUN``. Setting them to "true" will respectively show the
 commands to run before running them or skip running the commands.
 
-Note that you need to run pre-commit with --verbose command to get the output regardless of the status
+Note that you need to run prek with --verbose command to get the output regardless of the status
 of the static check (normally it will only show output on failure).
 
 Printing the commands while executing:
 
 .. code-block:: bash
 
-     VERBOSE="true" pre-commit run --verbose ruff
+     VERBOSE="true" prek --verbose ruff
 
 Just performing dry run:
 
 .. code-block:: bash
 
-     DRY_RUN="true" pre-commit run --verbose ruff
+     DRY_RUN="true" prek --verbose ruff
```

Comment:
`VERBOSE` and `DRY_RUN` don't work with prek for now, and I doubt them ever worked in `pre-commit` either(I can't find them in the docs or code). Do they comes from the `breeze static-checks` wrapper?

Reply:
Yes - they are breeze-specific variables. This is quite "deep" and already largely not-needed chapter. I removed it entirely.